### ISSUE TITLE
`[[core.data.variable]]` tags cannot be used in more than one config file

### ISSUE THREAD

**Issue by jacobcook1995** [ImperialCollegeLondon/virtual_ecosystem#234]
**Describe the bug**
`[[core.data.variable]]` tags cannot be used across multiple toml config files without causing `ConfigurationError: Duplicated entries in config files: core.data.variable` to be raised.

**To Reproduce**

1. Create a folder of config files with multiple files that define `[[core.data.variable]]` tags
2. From the terminal call `vr_run path/to/config/folder`
3. See error

**Expected behavior**
We probably want to be able to define data paths across multiple files, rather than forcing everyone to work with a single file. If we can't find a work around for this then it should be documented.

**Comment by davidorme** [comment]
I don't think this is a bug. I think we'd typically have users define all the data sources in one configuration file that might be used with different model configurations. If models each define their own data sources, they should clash if duplicated. So this is docs, I think

**Comment by jacobcook1995** [comment]
Probably should make clear that this isn't a clash where 2 files both define data sources for the same variable. This is a clash where more than one file defines (any) data sources. So, all datasources will have to be defined within a single toml file.

**Comment by vgro** [comment]
I've added a note to the documentation, the PR will show if I missed something. Maybe we need a new issue to actually fir the problem of not being able to use the functionality across `config` files?

**Comment by davidorme** [comment]
> Probably should make clear that this isn't a clash where 2 files both define data sources for the same variable. This is a clash where more than one file defines (any) data sources. So, all datasources will have to be defined within a single toml file.

I think this _is_ a bug then. Identical variable names should be an error, but not having data in different files.
